packages/VariantIcon/index.tsx

```diff
@@ -6,22 +6,24 @@ import { Variant } from '@welcome-ui/utils'
 
 import * as S from './styles'
 
+type Size = 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'xxl' | number | string
```

Comment:
The size could be any string? "test" is acceptable for instance?

Reply:
Indeed it should be removed 👍

Reply:
🔥